app/models/case_attachment_upload_group.rb

```diff
@@ -38,6 +38,6 @@ def <=>(other)
 private
 
   def get_time(upload_group)
-    Time.find_zone("Etc/UTC").parse(upload_group).in_time_zone
```

Comment:
Is `Time.find_zone` required, maybe just `Time.zone.parse` will work?

I think it would be good to add some tests for the `date_time` method which test times in in GMT and BST.